index.html

```diff
@@ -375,11 +375,30 @@ <h2>Contact Me</h2>
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
-        <form>
-          <input type="text" id="username" value="Yeremias" required maxlength="30" />
-          <input type="email" id="email" value="natajayanj14@gmail.com" required/>
-          <textarea name="details" id="detail" placeholder="Write your message here" required maxlength="500"></textarea>
-          <button class="get-in-touch" id="btn" type="submit">
+        <form action="https://formspree.io/f/mqknvnyz" method="post">
+          <input
+            type="text"
+            id="username"
+            required
+            maxlength="30"
+            placeholder="Text input"
+            name="userName"
+          />
+          <input
+            type="email"
+            id="email"
+            required
+            placeholder="vigehi2017@gmail.com"
+            name="userEmail"
+          />
+          <textarea
+            name="details"
+            id="detail"
+            placeholder="Write your message here"
+            required
+            maxlength="500"
+          ></textarea>
+          <button class="form-button go-projects" type="submit">
             Get in touch
           </button>
         </form>
```

Comment:
### Your Contact me section
![your-form](https://user-images.githubusercontent.com/93039359/178604702-08e20e12-6c49-49c5-a40e-b5d9bc3b28cb.png)

### Figma Template
![Figma-template](https://user-images.githubusercontent.com/93039359/178604713-ee9661a9-9331-4808-bb75-3d16c8ea8078.png)

- [ ] Your contact section is not following the Figma template. Please follow the Figma template :+1: